src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -281,14 +290,26 @@ const FilterForm = ({ filter, onCancel, onClearFilter, onSaveFilter, onSubmit, o
             options={dataLists.status}
           />
 
-          <CheckboxGroup
-            defaultValue={state.options.stadsdeel}
-            label="Stadsdeel"
-            name="stadsdeel"
-            onChange={onGroupChange}
-            onToggle={onGroupToggle}
-            options={dataLists.stadsdeel}
-          />
+          {configuration.useAreasInsteadOfStadsdeel && (
```

Comment:
If the feature flag is enabled, the districts should be available. Falling back to `stadsdelen` doesn't make sense in that case. We could do something else in case the data is not present, like showing a warning or something, but I think that's quite overkill.